### ISSUE TITLE
todo: make segmentation work with lmvr

### ISSUE THREAD

**Issue by j0ma** [j0ma/morph-seg#5]
Need to make LMVR segmentation work, using their hyperparameters.

**Comment by j0ma** [comment]
LMVR throws an error about `--lexicon-size`:

```
lmvr-train: error: unrecognized arguments: --lexicon-size
```

**Comment by j0ma** [comment]
> lmvr-train: error: unrecognized arguments: --lexicon-size

this can be fixed by installing LMVR in a Python 2.7 virtual environment (using `conda`)
that does NOT have vanilla flatcat installed. otherwise the paths will get mangled.

another thing that created problems was the perl snippet:

```
perl -pe 's/\n/ /g' | \
perl -pe 's/  /\n/g'
```  
which ended up getting rid of all newlines in the segmentation file.

removing the perl snippet caused the example script in `./src` to finally work. now just need to create model binaries and package it all.

**Comment by j0ma** [comment]
update: this works now, and models trained with parameters from the ![author's example script](https://github.com/d-ataman/lmvr/blob/master/examples/example-train-segment.sh) are in `bin`.
